wrangler.toml

```diff
@@ -0,0 +1,7 @@
+name = "rust"
```

Comment:
let's maybe call this `rust-worker` to not confuse with `type`

Reply:
The convention for templates is actually to have this be the same as the template name without "worker" because it'd get really redundant.  Also, think to look at an account with all the scripts names ending in worker